### PR TITLE
Add test coverage to JavadocTagContinuationIndentation check. #1308

### PR DESCRIPTION
**Pull request by mkordas**
Before:
image

After:
image

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java [hunk 1]
```diff
@@ -82,8 +82,7 @@ public void visitJavadocToken(DetailNode ast) {
             if (textNode != null && textNode.getType() == JavadocTokenTypes.TEXT
                     && textNode.getChildren().length > 1) {
                 final DetailNode whitespace = JavadocUtils.getFirstChild(textNode);
-                if (whitespace.getType() == JavadocTokenTypes.WS
```
Comment (mkordas) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java]: at this point `vwhitespace.getType() == JavadocTokenTypes.WS` always returns `true`

**Comment by mkordas** [comment]
I wrote `warn for offset 3 and 4, as indentation is 2` - it means that we expect warning if offset is set to 3 or 4, as the indentation of current line is 2.
I all other cases indentation is 3, so we expect warning only if offset is set to 4.

**Comment by romani** [comment]
I still do not understand that, please find me in chat for discussion.
In a comment we need to write what is actual, and do not write what possible could be.

**Comment by mkordas** [comment]
In comment there is information, that this line will cause warnings for two different test cases - the standard one with offset 4 and custom one with offset 3.

I'm waiting on chat.

**Comment by mkordas** [comment]
Separate input file has been added for clarity.

**Pull request merged**